src/v/coproc/script_context.cc

```diff
@@ -153,12 +186,36 @@ script_context::process_send_write(rpc::transport* t) {
     co_return ss::stop_iteration::no;
 }
 
+ss::future<errc> script_context::start_processing_ntp(
+  const model::ntp& ntp, read_context&& rctx) {
+    auto found = _routes.find(ntp);
+    if (found != _routes.end()) {
+        vassert(
+          rctx.input == found->second->rctx.input,
+          "Wanting to swap a different ctx for same input ntp");
+    }
+    auto [itr, _] = _updates.emplace(
+      ntp, std::make_unique<insert_update>(std::move(rctx)));
```

Comment:
You got `&&` for rctx, but use move inside
Maybe change `read_context&& rctx` to `read_context rctx`

Reply:
Thats fine though right? You can move rvalues